CMSIS/Documentation/Doxygen/Core_A/src/mainpage.md

```diff
@@ -39,15 +41,17 @@ CMSIS supports a selected subset of [Cortex-A processors](https://www.arm.com/pr
 **Cortex-A Technical Reference Manuals**
 
 The following Technical Reference Manuals describe the various Arm Cortex-A processors:
+
  - [Cortex-A5](https://developer.arm.com/documentation/ddi0433) (Armv7-A architecture)
  - [Cortex-A7](https://developer.arm.com/documentation/ddi0464) (Armv7-A architecture)
  - [Cortex-A9](https://developer.arm.com/documentation/100511) (Armv7-A architecture)
 
 ## Tested and Verified Toolchains {#tested_tools_sec}
 
-The \ref templates_pg supplied by Arm have been tested and verified with the following toolchains:
- - Arm: Arm Compiler 5.06 update 7
- - Arm: Arm Compiler 6.16
- - Arm: Arm Compiler 6.6.4
- - GNU: GNU Arm Embedded Toolchain 10-2020-q4-major (10.2.1 20201103)
- - IAR: IAR ANSI C/C++ Compiler for Arm 8.20.1.14183
+The \ref templates_pg delivered with this CMSIS-Core release have been tested and verified with the following toolchains:
+
+ - Arm Compiler for Embedded 6.21
```

Comment:
I think AC 6.20 is used because 6.21 is not available via vcpkg / artifacts.keil.arm.com yet.

Reply:
The environment used for testing is pinned here:
https://github.com/ARM-software/CMSIS_6/blob/main/CMSIS/CoreValidation/Project/vcpkg-configuration.json